CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Pricing
 
-* New per trip pricing option. For more information, see the “[Pricing](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/)” guide ([#3147](https://github.com/mapbox/mapbox-navigation-ios/pull/3147))
+* New per trip pricing option. For more information, see the “[Pricing](https://docs.mapbox.com/ios/beta/navigation/guides/pricing/)” guide ([#3147](https://github.com/mapbox/mapbox-navigation-ios/pull/3147), [#3338](https://github.com/mapbox/mapbox-navigation-ios/pull/3338))
 * `MBXNavigationBillingMethod` is no longer supported. ([#3147](https://github.com/mapbox/mapbox-navigation-ios/pull/3147))
 
 ### Packaging
```

Comment:
It's interesting why don't we mention `MapboxSpeech` as a dependency in the changelog? Should we start doing it now?

Reply:
Probably not. @1ec5?